tests/src/helpers.c

```diff
@@ -341,18 +341,26 @@ int mbedtls_test_ascii2uc(const char c, unsigned char *uc)
     return 0;
 }
 
```

Comment:
(Out of scope here, but please consider a follow-up) Several functions in this file are missing `static`, e.g. `mbedtls_test_set_line1`. I guess every function where the caller must hold the mutex should be `static`.

Also, it would help if those function's names reflected this. For example it's surprising that `mbedtls_test_set_line1` and `mbedtls_test_get_line1` have different mutex policies.